### PR TITLE
add brewery

### PR DESCRIPTION
**Pull request by jhbha**

### CODE REVIEW

**Comment by rk295** [comment]
I feel compelled to insist it must contain [Apex](https://apex.beer/). 

Also would you mind lower casing them pls?

**Comment by jhbha** [comment]
No problem 🤣😂 - I got the list from Wikipedia I think so can’t vouch for the accuracy!Sent from my iPhoneOn 28 Nov 2022, at 21:33, Robin Kearney ***@***.***> wrote:﻿
I feel compelled to insist it must contain Apex.
Also would you mind lower casing them pls?

File: lib/data/breweries.txt [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+plain
+poachers
+portobelloco
+potbelly
+potton
+prescott
+pressuredrop
+prospect
+purity
+q
```
Comment (rk295) [inline lib/data/breweries.txt:501]: wtf

**Comment by jhbha** [comment]
Seems legit - https://www.facebook.com/qbrewuk/Sent from my iPhoneOn 29 Nov 2022, at 16:58, Robin Kearney ***@***.***> wrote:﻿
@rk295 commented on this pull request.



In lib/data/breweries.txt:
> +phoenix
+pictish
+piedbull
+pig-porter
+pilgrim
+pinnora
+plain
+poachers
+portobelloco
+potbelly
+potton
+prescott
+pressuredrop
+prospect
+purity
+q

wtf

**Pull request merged**